lms/static/js/spec/dashboard/track_events_spec.js

```diff
@@ -0,0 +1,95 @@
+;(function (define) {
```

Comment:
Nitpick: the semicolon at the beginning of this file isn't necessary. Nice work resolving the quality issue.